docs/guides/api.en.md

```diff
@@ -212,6 +297,26 @@
     | :--- | :--- |
     | `String` | The content of the code if found. |
 
+??? info "SetStringGMLInFile `method`"
```

Comment:
Maybe flag this as legacy ?

Reply:
Yep, we do want people to use the chained methods.